src/pages/cli/reference/diagnose.mdx

```diff
@@ -0,0 +1,53 @@
+export const meta = {
+  title: `Diagnose`,
+  description: `More information about feature flags in Amplify CLI`,
```

Comment:
Should description say feature flags or something like `More information about diagnose command in Amplify CLI`?